cmd/ns-eth/main.go

```diff
@@ -20,25 +20,38 @@ import (
 )
 
 var (
-	log       = slf.WithContext("main")
+	log = slf.WithContext("ns-eth")
+
+	// Set externaly during build
 	branch    string
 	commit    string
+	lasttag   string
 	buildtime string
 
 	memprofile = flag.String("memprofile", "", "write memory profile to `file`")
 )
 
 var globalOpt = ns.Configuration{
-	Name: "eth-node-service",
+	CanaryTest: false,
+	Name:       "eth-node-service",
 }
 
 func main() {
-	config.ReadGlobalConfig(&globalOpt, "multy configuration")
-	log.Infof("CONFIGURATION=%+v", globalOpt)
-
+	log.Info("============================================================")
+	log.Info("Node service ETH starting")
 	log.Infof("branch: %s", branch)
 	log.Infof("commit: %s", commit)
 	log.Infof("build time: %s", buildtime)
+
+	log.Info("Reading configuration...")
+	config.ReadGlobalConfig(&globalOpt, "multy configuration")
+	log.Infof("CONFIGURATION=%+v", globalOpt)
```

Comment:
according with https://github.com/Multy-io/Multy-back/issues/97
don't print configuration or globalOpt.Database.Password = "***"

Reply:
Not all fixes from master/release were ported yet, and this line here is just something that existed before, just rearranged.

Reply:
k